go/adbc/driver/internal/driverbase/database.go

```diff
@@ -136,4 +201,218 @@ func (db *database) SetLogger(logger *slog.Logger) {
 	}
 }
 
+func (base *database) InitTracing(ctx context.Context, driverName string, driverVersion string) error {
+	return base.Base().InitTracing(ctx, driverName, driverVersion)
+}
+
+func (base *DatabaseImplBase) InitTracing(ctx context.Context, driverName string, driverVersion string) (err error) {
+	fullyQualifiedDriverName := driverNamespace + "." + driverName
+
+	getExporterName := sync.OnceValue(func() string {
+		return os.Getenv(otelTracesExporter)
+	})
```

Comment:
this needs to get hoisted out of this function, otherwise you're creating a new `sync.OnceValue` on every call. defeating the purpose. This should be file-level